### PR TITLE
obj: improve log messages for truncated object uploads

### PR DESCRIPTION
**Pull request by fvennetier**
Previous messages were `Client disconnected without sending last chunk`.
Now they give details about the expected size and the actual size received by the gateway: `Truncated input (763831 bytes read, 8888000 bytes expected)`.

### CODE REVIEW

File: oioswift/proxy/controllers/obj.py [hunk 1]
```diff
@@ -595,7 +599,8 @@ def _link_object(self, req):
         except exceptions.SourceReadError:
```
Comment (murlock) [inline oioswift/proxy/controllers/obj.py]: should be `except exceptions.SourceReadError as err:` ?

File: oioswift/proxy/controllers/obj.py [hunk 2]
```diff
@@ -595,7 +599,8 @@ def _link_object(self, req):
...
             req.client_disconnect = True
             self.app.logger.warning(
-                _('Client disconnected without sending last chunk'))
+                _('Client disconnected without sending last chunk') + (
```
Comment (murlock) [inline oioswift/proxy/controllers/obj.py:602]: I don't know if a "link" operation can raise a `SourceReadError`

**Review by murlock — APPROVED** [review]

File: oioswift/proxy/controllers/obj.py (hunk 1 above)
Comment (murlock) [reply oioswift/proxy/controllers/obj.py]: already fix during my review ! :+1:

**Review by AymericDu — APPROVED** [review]

File: oioswift/proxy/controllers/obj.py (hunk 2 above)
Comment (fvennetier) [reply oioswift/proxy/controllers/obj.py]: Indeed, `SourceReadError` makes no sense in this context.

**Pull request merged**